lib/tinyset.rs

```diff
@@ -17,19 +17,25 @@ pub struct tinyset_Set64<T: u64set::Fits64 + Ord> {
     pub x: u64set::Set64<T>
 }
 
-/* This is needed so we can support for-loops over `Set64`'s
+/* This is needed so we can support for-loops over `Set64`'s.
+ *
+ * IMPORTANT: We iterate over Set64 by converting it into a sorted vector.
```

Comment:
hopefully this conversion is itself deterministic

Reply:
I think this is the case, since the element type must be totally ordered (`Ord`)